Tasks/AzureRmWebAppDeployment/azurermwebappcontainerdeployment.ts

```diff
@@ -0,0 +1,65 @@
+import tl = require('vsts-task-lib/task');
+import util = require('util');
+
+var azureRESTUtility = require ('azurerest-common/azurerestutility.js');
+var parameterParser = require("./parameterparser.js").parse;
+
+export async function deployWebAppImage(endPoint, resourceGroupName, webAppName) {
+    var startupCommand = tl.getInput('StartupCommand', false);
+    var appSettings = tl.getInput('AppSettings', false);
+    var imageSourceAndTag;
+
+    // Construct the image
+    var dockerNamespace = tl.getInput('DockerNamespace', true);
+    var dockerRepository = tl.getInput('DockerRepository', true);
+    var dockerImageTag = tl.getInput('DockerImageTag', false);
+
+    if(dockerImageTag) {
+        imageSourceAndTag = dockerNamespace + "/" + dockerRepository + ":" + dockerImageTag;          
+    } else {
+        imageSourceAndTag = dockerNamespace + "/" + dockerRepository;
+    }
+
+    if(imageSourceAndTag)
+    {
+        tl.debug("Deploying the image " + imageSourceAndTag + " to the webapp " + webAppName);
+        
+        appSettings = appSettings ? appSettings.trim() : "";        
```

Comment:
Can user provide DOCKER_CUSTOM_IMAGE_NAME in appSettings via Task input? Should we error out if present?